vhost-device-vsock/src/vhu_vsock_thread.rs

```diff
@@ -441,6 +442,13 @@ impl VhostUserVsockThread {
                             if cnt > 0 {
                                 conn.fwd_cnt += Wrapping(cnt as u32);
                                 conn.rx_queue.enqueue(RxOps::CreditUpdate);
+                            } else if conn.tx_buf.is_empty() {
```

Comment:
Can we just check if `cnt == 0`?

IIUC `flush_to` returns `Ok(0)` if `tx_buf` is empty,